### PR TITLE
pass zoom level to Mapnik to allow usage of the zoom variable

### PR DESCRIPTION
**Pull request by nebulon42**
Ref https://github.com/mapbox/carto/issues/269

### CODE REVIEW

File: src/back/Tile.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ Tile.prototype.render = function (project, map, cb) {
     this.setupBounds();
     map.zoomToBox(this.projection.forward([this.minX, this.minY, this.maxX, this.maxY]));
     var im = new mapnik.Image(this.height, this.width);
-    map.render(im, {scale: project.mml.scale}, cb);
+    map.render(im, {scale: project.mml.scale, variables: {zoom: this.z}}, cb);
```
Comment (yohanboniface) [inline src/back/Tile.js]: Should we also pass the variable line 41?

Comment (nebulon42) [inline src/back/Tile.js]: Probably a good idea, but I haven't setup a vector source, so cannot test. Related: what about the scale factor for vector tiles?

Comment (yohanboniface) [inline src/back/Tile.js]: I think there is no need given it's already vector data, so scalable consumer side, but I may miss something.
I'll merge as is, we'll see about line 41 is the need arise.

Comment (springmeyer) [inline src/back/Tile.js]: I vaguely remember an unresolved by where `variables` was not handled in the vector tile pathway. Create a ticket a node-mapnik if you see this. Can't recall if I fixed it...

**Pull request merged**